hide snippet nav when demo has a single snippet

miguelcobain opened the pull request:
Only shows snippet nav when more than one snippet is present. Looks like this:

![screen shot 2019-01-31 at 20 43 32](https://user-images.githubusercontent.com/631691/52083997-ee03e480-2598-11e9-9ad0-396afe6192f4.png)

samselikoff commented:
looks awesome! Any chance we could add a 1px border to the top, to differentiate the two sections?

miguelcobain commented:
Ok, updated.
Added top border with the same background as the nav. Only added when the nav is hidden.

![screen shot 2019-01-31 at 20 54 21](https://user-images.githubusercontent.com/631691/52084787-c4e45380-259a-11e9-8b8f-b6ef9c0a25f6.png)
